### ISSUE TITLE
Add tagger support for sort orders and original release dates

### ISSUE THREAD

**Issue by unikitty37** [lidarr/Lidarr#2105]
**Is your feature request related to a problem? Please describe.**
Lidarr's MusicBrainz tagging does not appear to update Artist Sort Order, Album Artist Sort Order, Original Release Date, and Original Year tags.

**Describe the solution you'd like**
If those four tags could be added to the list of tags that Lidarr will update from MusicBrainz,  it would make life a lot easier for people wanting to sort by artist surname or album release year. 

**Describe alternatives you've considered**
Manually loading every album into Picard and then saving it is the only solution I can think of, which gets tedious fast 😁 

**Additional context**
After importing [this album](https://musicbrainz.org/release/446f2b31-cb00-456d-bcaa-24f71884e34c?tport=8000) into Lidarr and clicking "Preview Retag", Lidarr says "Success! My work is done, no files to retag."

If I then load the album into Picard, it shows the following changes could be applied:

![image](https://user-images.githubusercontent.com/139183/112011388-23287380-8b20-11eb-8b36-dfffc563417d.png)

I can live without Scripts, Catalogue Number and Artists, but the two Sort Order tags stop the artist appearing in the correct place in my Kodi library, and the missing Original Release Date/Year tags stop the album appearing in the correct place in the list of that artist's albums: a 1996 remaster of Jethro Tull's Aqualung — originally released in 1971 — shows after much more recent albums because it gets sorted by the remaster's release date rather than the original album's release date, despite having Kodi configured to use the original release date.

**Comment by sdfg2** [comment]
Yes, very much this.  (And amusingly I noticed it with Jethro Tull as well, plus all the other prog bands that did re-releases).

Please also add these fields to the file renamer.

**Comment by swills1** [comment]
Catalogue number and original release date would be very helpful. I use original release date to determine if a release is a is a reissue, and I assign a cat number to everything so I know exactly what release I have. Right now the only option would be to trigger a script that kicks off Beets. But I'd have to devise a way for Beets to grab the right release based on Cat number.
